html/changelogs/DeityLink_6032.yml

```diff
@@ -0,0 +1,20 @@
+author: Deity Link
+delete-after: true
+changes:
```

Comment:
hit the high points and make em interested, give them the details in the original post of the PR and the thread